ktor-client/ktor-client-core/common/src/io/ktor/client/content/ObservableContent.kt

```diff
@@ -18,7 +18,12 @@ import kotlin.coroutines.*
  * @param bytesSentTotal number of transmitted bytes
```

Comment:
Isn't it better to move kDoc for parameters `bytesSentTotal` and `contentLength` in kDoc for `fun onProgress`

Reply:
Maybe, but I'll rather do it separately, because this PR is already too big